WMass/python/plotter/w-helicity-13TeV/mergeCardComponentsAbsY.py

```diff
@@ -298,7 +298,7 @@ def putEffStatHistos(infile,regexp,charge):
                         bin = l.split()[1]
                         binn = int(bin.split('_')[-1]) if 'Ybin_' in bin else -1
                     rootfiles_syst = filter(lambda x: re.match('\S+{base}_sig_(pdf\d+|muR\S+|muF\S+|alphaS\S+|mW\S+)\.input\.root'.format(base=basename),x), allfiles)
-                    if ifile==0:
+                    if 'Z_' in f:
```

Comment:
so that we don't depends on the sorting of the nominal files (bkg and data can become the 0th)